packages/react-components/src/organisms/SharedResearchList.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import css from '@emotion/css';
+import { ResearchOutputResponse } from '@asap-hub/model';
+import { sharedResearch } from '@asap-hub/routing';
+
+import { Card, Anchor, TagLabel, Headline2 } from '../atoms';
+import { steel } from '../colors';
+import { ExternalLink } from '../molecules';
+import { paddingStyles } from '../card';
+
+const containerStyles = css({
+  margin: 0,
+  padding: 0,
+  listStyle: 'none',
+});
+const itemStyles = css({
+  borderBottom: `1px solid ${steel.rgb}`,
+  display: 'grid',
+  gridTemplateRows: 'auto 12px auto',
+  '* ~ div:last-of-type': {
```

Comment:
I assume you mean this? 
```suggestion
  '&:last-of-type': {
```